pennylane/ops/projection.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the available built-in quantum projection operations
+supported by PennyLane.
+"""
+
+import numpy as np
+
+from pennylane.operation import AnyWires, Projection
+
+
+class Measure(Projection):
+    r"""Measure(wires)
+    A measurement in the computational basis.
+
+    **Details:**
+
+    * Number of wires: Any
+    * Number of parameters: 0
+    * Gradient recipe: None
+
+    Args:
+        wires (Sequence[int] or int): the wire(s) the operation acts on
+
+    .. UsageDetails::
+
+        It is important to note when using the `Measurement` operation that
+        one **applies** it to the wire one wishes to measure, and then returns
+        one of the PennyLane measurement functions.
```

Comment:
```suggestion
        The `Measurement` operation is applied to the wire we
        want to measure. The return of a circuit is then evaluated on
        the output state conditioned on the given measurement outcome.
        This will depend on the random outcome of the measurement,
        so expectation values should be obtained over several shots. 
```